Dockerfile

```diff
@@ -1,3 +1,11 @@
+ARG DEBIAN_FRONTEND=noninteractive
+ARG GITHUB_WORKSPACE=/build
+ARG ZIG_PATH=${GITHUB_WORKSPACE}/zig
+# Directory extracts to "bun-webkit"
+ARG WEBKIT_DIR=${GITHUB_WORKSPACE}/bun-webkit 
```

Comment:
Is `ARG` per file and not per image?

Reply:
Sorry, am seeing this now !
I think it's okay to have it like that, is it possible to make some tests too ?